Library/Homebrew/os.rb

```diff
@@ -13,6 +13,10 @@ def self.linux?
     RbConfig::CONFIG["host_os"].include? "linux"
   end
 
+  def self.kernel_version
+    @kernel_version ||= Version.new(`uname -r`.strip)
```

Comment:
- What if `uname -r` fails? (should probably `raise`)
- `Utils.popen_read` might be a nicer fit here.

Reply:
Also: could this get a unit test.